fdbclient/BlobWorkerInterface.h

```diff
@@ -43,14 +43,24 @@ struct BlobWorkerInterface {
 
 	void initEndpoints() {}
 	UID id() const { return myId; }
+	// TODO: why do we get addresses like this? getEndpoint gives u current machines addr?
```

Comment:
I think this is for historical/compatibility reasons with LoadBalance not knowing what type this interface is?

Reply:
Thanks - this was more-so a question for my personal knowledge :)